ui/app/components/pages/add-token/add-token.component.js

```diff
@@ -28,27 +28,27 @@ class AddToken extends Component {
     identities: PropTypes.object,
   }
 
-  constructor (props) {
-    super(props)
+  static defaultProps = {
+    pendingTokens: {},
+  }
 
-    this.state = {
-      customAddress: '',
-      customSymbol: '',
-      customDecimals: 0,
-      searchResults: [],
-      selectedTokens: {},
-      tokenSelectorError: null,
-      customAddressError: null,
-      customSymbolError: null,
-      customDecimalsError: null,
-      autoFilled: false,
-      displayedTab: SEARCH_TAB,
-    }
+  state = {
+    customAddress: '',
+    customSymbol: '',
+    customDecimals: 0,
+    searchResults: [],
+    selectedTokens: {},
+    tokenSelectorError: null,
+    customAddressError: null,
+    customSymbolError: null,
+    customDecimalsError: null,
+    autoFilled: false,
+    displayedTab: SEARCH_TAB,
   }
 
   componentDidMount () {
     this.tokenInfoGetter = tokenInfoGetter()
-    const { pendingTokens = {} } = this.props
```

Comment:
I think we need this to guard against `pendingTokens` being undefined on line 59

Reply:
oh, I guess that is handled by the addition of `defaultProps` above